pkg/minikube/node/start.go

```diff
@@ -669,3 +675,18 @@ func prepareNone() {
 		exit.Message(reason.HostHomeChown, "Failed to change permissions for {{.minikube_dir_path}}: {{.error}}", out.V{"minikube_dir_path": localpath.MiniPath(), "error": err})
 	}
 }
+
+// updateCoreDNS adds host name and IP record to the DNS by updating CoreDNS's ConfigMap.
+// ref: https://coredns.io/plugins/hosts/
+func updateCoreDNS(runner command.Runner, name, ip string, cc config.ClusterConfig) error {
```

Comment:
More specific name may be? `addCoreDNSEntry`?

Reply:
renamed to addCoreDNSEntry :)